packages/config/__tests__/ProfileInfo.OldProfiles.test.ts

```diff
@@ -0,0 +1,481 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import * as path from "path";
+import { ProfileInfo } from "../src/ProfileInfo";
+import { IProfOpts } from "../src/doc/IProfOpts";
+import { ProfInfoErr } from "../src/ProfInfoErr";
+import { ProfLocType } from "../src/doc/IProfLoc";
+import { IProfileSchema, ProfileIO } from "../../profiles";
+
+const testAppNm = "ProfInfoApp";
+const testEnvPrefix = testAppNm.toUpperCase();
+const profileTypes = ["zosmf", "tso", "base", "dummy"];
+
+function createNewProfInfo(newDir: string, opts?: IProfOpts): ProfileInfo {
+    // create a new ProfileInfo in the desired directory
+    process.chdir(newDir);
+    const profInfo = new ProfileInfo(testAppNm, opts);
+    jest.spyOn((profInfo as any).mCredentials, "isSecured", "get").mockReturnValue(false);
+    return profInfo;
+}
+
+describe("ProfileInfo tests", () => {
+
+    const tsoName = "tsoProfName";
+    const testDir = path.join(__dirname, "__resources__");
+    const homeDirPath = path.join(testDir, testAppNm + "_home");
+    const homeDirPathTwo = path.join(testDir, testAppNm + "_home_two");
+    const homeDirPathThree = path.join(testDir, testAppNm + "_home_three");
+    const four = 4;
+    let origDir: string;
+
+    beforeAll(() => {
+        // remember our original directory
+        origDir = process.cwd();
+    });
+
+    beforeEach(() => {
+        // set our desired app home directory into the environment
+        process.env[testEnvPrefix + "_CLI_HOME"] = homeDirPath;
+    });
+
+    afterAll(() => {
+        // ensure that jest reports go to the right place
+        process.chdir(origDir);
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    describe("Old-school Profile Tests", () => {
```

Comment:
Thanks for splitting the old-school and team config tests into 2 separate files 👍 Is this `describe` block still necessary, or is it an extra layer that can be removed if we rename the top-level `describe` to "Old-school Profile Tests"?